<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
      <item>
       <widget class="QWidget" name="widgetHeader" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelLogo">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTitle">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelAppMode">
             <property name="text">
              <string>Light Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkAppMode">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSetting">
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnUserName">
             <property name="text">
              <string>CLOCK IN</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
        <item>
         <widget class="QWidget" name="widgetNav" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>18</number>
             </property>
             <item>
              <widget class="QLCDNumber" name="lcdDateTime">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="digitCount">
                <number>19</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUserList">
               <property name="text">
                <string>User List</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUserManagement">
               <property name="text">
                <string>User Management</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnTaskManagement">
               <property name="text">
                <string>Task Management</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAccountabilityReports">
               <property name="text">
                <string>Accountability Reports</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnUserList</tabstop>
  <tabstop>btnUserManagement</tabstop>
  <tabstop>btnTaskManagement</tabstop>
  <tabstop>btnAccountabilityReports</tabstop>
  <tabstop>btnUserName</tabstop>
 </tabstops>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
